docs/apis/management/user_management.yaml

```diff
@@ -224,6 +224,88 @@ paths:
                   message:
                     type: string
                     description: Error message for an unprocessable entity
+  /roles:
+    get:
+      summary: Get roles
+      description: Retrieve roles data for account with list of users for each role
+      tags:
+        - Managing roles
+      responses:
+        "200":
+          description: Successful response containing the list of roles with capabilities
+            and users for each role
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  roles:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                          description: The display name of the role
+                        type:
+                          type: string
+                          enum:
+                            - predefined
+                            - custom
+                          description: Describes whether the role is custom or pre-defined
+                        capabilities:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              section:
```

Comment:
Is there a close list of sections?

Reply:
The question is if you would like to add an enum here?

Reply:
No - an enum here will force us to update this every time we add a new section.